### ISSUE TITLE
Send reviewer feedback to mail address

### ISSUE THREAD

**Issue by krrisis** [Giveth/giveth-dapp#829]
After a Milestone has been reviewed (rejected or returned) it would be good to get a mail with the added feedback. Now you only see that feedback if you actually go back into the Milestone, otherwise you never see the written feedback you received.

**Comment by GriffGreen** [comment]
Yeah cool! I would love to see the email stuff get tackled... there are a lot of improvements that could happen there

**Comment by horus2021** [comment]
I should check out this issue. I think it has been fixed.
